chart/tests/with-datadog-service-label/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Configuration settings for the joy-generator instance
+env:
+  GH_APP_ID: "123"
+  GH_APP_INSTALLATION_ID: "666"
+
+  # The HTTPS Git URL of your joy catalog
+  CATALOG_URL: "https://github.com/example/foobar.git"
+
+secretEnv:
+  values:
+    PLUGIN_TOKEN: "@very!l0ngands3curet0ken"
+
+# The GitHub app configuration.
+# Required if githubToken is not set.
+# The GitHub application private key (PEM format)
+githubAppPrivateKey: |
+  FOOBARLOREMISPUM
+
+image:
+  tag: 0.1.2
+
+datadogServiceLabel: joy-generator-chart-test
```

Comment:
this is a bit opinionated on having datadog. 

I would go with a more generic approach of having an `additionalLabels` or, more specifically a `podLabels` field which is a map of labels & values. This is the approach most charts use.